cypress/component/UserProfile/user_profile.spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-undef */
+
+import {mount} from 'cypress/react';
+import React from 'react';
+import {Storage} from '../../../src/libs/storage';
+import {User} from '../../../src/libs/ajax/User';
+import {Institution} from '../../../src/libs/ajax/Institution';
+import UserProfile from '../../../src/pages/user_profile/UserProfile';
+
+const duosUser = {
+    isSigningOfficial: false,
+};
+
+describe('User Profile', () => {
+  it('Renders the user profile page', () => {
+    cy.stub(Storage, 'getCurrentUser').returns(duosUser);
+    mount(<UserProfile/>);
+    cy.get('h2').should('contain', 'Your Profile');
+  });
```

Comment:
This test is making live calls to consent. I think you need to stub the same calls as you have in the next test:
```
    cy.stub(Institution, 'list').returns([]);
    cy.stub(User, 'getMe').returns(duosUser);
    cy.stub(User, 'getApprovedDatasets').returns([]);
    cy.stub(User, 'getAcknowledgements').returns({});
```

Reply:
Alternatively, the duplicate stubs could be in a `beforeEach`.